tests/test_mig_shared_install.py

```diff
@@ -56,7 +57,9 @@ def create_dummy_gpwnam(pw_uid, pw_gid):
 
 
 def noop(*args, **kwargs):
-    pass
+    if (len(args) > 0):
```

Comment:
Minor style: just use `if args:` for testing non-empty list args as per PEP8. I'll fix in the merge :)

Reply:
Ahh yes, as mentioned elsewhere this a holdover from different coercion semantics in JS.